### PR TITLE
[Pages] Sphinx framework guide fix

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/pages/src/content/framework-guides/deploy-a-sphinx-site.md [hunk 1]
```diff
@@ -55,7 +54,7 @@ $ pipenv --version
 pipenv, version 2021.5.29
 ```
 
-### Creating a Sphinx project directory
+### Installing Sphinx
```
Comment (kristianfreeman) [inline products/pages/src/content/framework-guides/deploy-a-sphinx-site.md:57]: In this case, the initial header is correct - can we update in the style we prefer, but keep the idea of "Creating a Sphinx project directory"?

**Review by kristianfreeman — CHANGES_REQUESTED** [review]

File: products/pages/src/content/framework-guides/deploy-a-sphinx-site.md (hunk 1 above)
Comment (deadlypants1973) [reply products/pages/src/content/framework-guides/deploy-a-sphinx-site.md:57]: I edited it according to this https://github.com/cloudflare/cloudflare-docs/pull/1876#pullrequestreview-725827484

Comment (deadlypants1973) [reply products/pages/src/content/framework-guides/deploy-a-sphinx-site.md:57]: It seems like "### Creating a Sphinx project directory" needs to go later in the tutorial

Comment (kristianfreeman) [reply products/pages/src/content/framework-guides/deploy-a-sphinx-site.md:57]: I think we install sphinx on L100, maybe @jeffh-cloudflare can clarify what he has in mind here?

Comment (jeffh-cloudflare) [reply products/pages/src/content/framework-guides/deploy-a-sphinx-site.md:57]: I think this is an issue of use of us not using consistent terminology (my bad):

There are three operations as part of installing/configuring Sphinx:

1. Create a directory on local filesystem used for managing the project
2. Installing sphinx with Pipenv
3. Running sphinx-quickstart - that creates the actual Sphinx Project (this is where I think the terminology goes sideways) :-)

Does that help? I apologize - the GitHub repository workflow is very confusing to me.

**Pull request closed without merging**